R/knit_print.R

```diff
@@ -7,13 +7,66 @@
 #'
 #' @keywords internal
 #' @noRd
-knit_print.fontawesome <- function(x, ...) {
+knit_print.fontawesome <- function(x, ..., options, inline = FALSE) {
 
   # nocov start
 
-  if (requireNamespace("knitr", quietly = TRUE)) {
-    knitr::knit_print(as.character(x), ...)
+  # get pandoc output format
+  to <- knitr::pandoc_to()
```

Comment:
If we want to remove the knitr version requirement
```suggestion
  to <- knitr::opts_knit$get("rmarkdown.pandoc.to")
```

but we can also add a version check. As you think it is best. Maybe the former ?

Reply:
I think using the suggested change might be better. Then we'd wouldn't have to 'force' someone to update rmarkdown/knitr. Much later on, the code could be refactored to use `knitr::pandoc_to()`.

@wch what's your take on this?

Reply:
Sure, that sounds good to me. Can you also add a comment about changing it to `pandoc_to()` in the future?

Reply:
Let's do this then. It will make it work for those with older version too. 

However, I don't see why **rmarkdown** needs to be updated though. 🤔 Last version should not be knitted for this to work. I'll check again.